src/accordionpane/example/index.ts

```diff
@@ -15,8 +15,9 @@ export class App extends WidgetBase<WidgetProperties> {
 	private _openKeys = new Set<string>();
 	private _theme: {};
 
-	themeChange(event: TypedTargetEvent<HTMLInputElement>) {
-		this._theme = event.target.checked ? dojoTheme : {};
+	themeChange(event: Event) {
+		const target: HTMLInputElement = event.target as any;
+		this._theme = target.checked ? dojoTheme : {};
```

Comment:
This looks functionally equivalent to the button index change, but uses slightly different code.

Reply:
Yeah, I wasn't consistent and should have been. I can clean that up next week.